error-budgets/SLOs-SLIs.md

```diff
@@ -0,0 +1,64 @@
+# Definitions of SLOs and SLIs for packit
+
+If you are not familiar with SLOs and SLIs, please check [README.md](./README.md) first.
+
+## How do others do it?
+
+The CKI team has a pretty comprehensive set of
+[SLOs](https://cki-project.org/docs/hacking/rfcs/cki-004-slo/#created-slos-based-on-the-feedback).
+Instead of taking the same route, let's start small and slowly add more and
+more objectives and indicators over time.
+
+Testing Farm team has [Service level
+expectations](https://docs.google.com/document/d/1iHK3v_tCK4w-F82ZvUEO3xHcNv78tkR31w3oF-zpi1Q/edit#heading=h.26gd46u8e15q)
+and a pretty slick
+[dashboard](http://grafana-latest-osci-infra.cloud.privileged.psi.redhat.com/d/VXwH27XMk/rhel-infrastructure-and-service-health?orgId=1).
+
+Cockpit also uses this framework and the team has [a nice dashboard as well](http://grafana-latest-osci-infra.cloud.privileged.psi.redhat.com/d/VgtdtiwGk/rhel-infrastructure-and-service-health?orgId=1).
+
+## Packit SLOs
+
+SLOs are based on [discussions with our stakeholders](users-expectations.md).
+
+Let's define job statuses first to get a clear understanding of these objectives:
+
+- successful run: a job finished and everything is awesome
+- failure: the job finished and wasn't successful: the PR needs to be fixed
+- error: infrastructure problems prevented the job to complete
+
+### 99% of builds finish with success/failure by 24 hours
+
+The most core functionality is to run COPR builds for PRs. We want to be sure
+those builds either pass or fail and no error interrupts the build process.
+
+The problem is that some builds take minutes and some hours so it's hard to
+design this objective to make it more proactive - that we'd know sooner that
+something is wrong.
```

Comment:
I don't think there is anything wrong with that. I don't expect this objective to be used for short term monitoring.